api/controllers/document.js

```diff
@@ -671,4 +676,56 @@ exports.protectedDelete = async function (args, res, next) {
     console.log("Error:", e);
     return Actions.sendResponse(res, 400, e);
   }
-};
\ No newline at end of file
+};
+
+exports.featureDocument = async function (args, res, next) {
+  try
+  {
+    if (args.swagger.params.docId && args.swagger.params.docId.value) 
+    {
+      let document = await mongoose.model('Document').findById(mongoose.Types.ObjectId(args.swagger.params.docId.value));
+      let project = await mongoose.model('Project').findById(mongoose.Types.ObjectId(document.project));
+      
+      if(project)
+      {
+        let featuredDocumentsCount = await mongoose.model('Document').count({ project: project._id, isFeatured: true }); 
+
+        // Move the magic number into a config
+        if(featuredDocumentsCount < 5)
```

Comment:
Can we move this into a config now?

Reply:
And should this be <= 5?

Reply:
Yep, I can move it into a config. I'll update shortly.

< 5 is correct in this case as the if sets the flag. <= would allow you to set 6 docs as featured per project